templates/jobs/job_edit.html

```diff
@@ -3,14 +3,23 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
-    <div class="container">
-        <div class="row">
-            <h1>New offer</h1>
-            <hr/>
-            <form method="POST" class="job-form">{% csrf_token %}
-                {{ form|crispy }}
-                <button type="submit" class="save btn btn-default">Save</button>
-            </form>
-        </div>
+<div class="container">
+    <div class="row">
+        <h1>New offer</h1>
+        <hr/>
+        <form method="POST" class="job-form">{% csrf_token %}
+            {% if form.show_overwrite %}
+            <button type="submit" class="save btn btn-default"
+                    name="overwrite">Overwrite
+            </button>
+            <button type="submit" class="save btn btn-default" name="keep">Keep old</button>
```

Comment:
"Cancel" rather than "Keep old"?  If you say "Keep Old", the entire add job form is discarded, not just the company website overwrite.